hash_practice/exercises.py

```diff
@@ -2,18 +2,50 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(m n log n)
+        Space Complexity: O(n)
     """

```

Comment:
👍 Correct.  If the words are limited in size you can define this as O(m) time complexity where m is the number of words.